src/Microsoft.Diagnostics.Monitoring/Configuration/HttpRequestSourceConfiguration.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using Microsoft.Diagnostics.NETCore.Client;
+
+namespace Microsoft.Diagnostics.Monitoring.Configuration
+{
+    public sealed class HttpRequestSourceConfiguration : MonitoringSourceConfiguration
+    {
+        private const string DiagnosticFilterString = "\"" +
+                "Microsoft.AspNetCore/Microsoft.AspNetCore.Hosting.HttpRequestIn.Start@Activity1Start:-" +
+                    "Request.Scheme" +
+                    ";Request.Host" +
+                    ";Request.PathBase" +
+                    ";Request.QueryString" +
+                    ";Request.Path" +
+                    ";Request.Method" +
+                    ";ActivityStartTime=*Activity.StartTimeUtc.Ticks" +
+                    ";ActivityParentId=*Activity.ParentId" +
+                    ";ActivityId=*Activity.Id" +
+                    ";ActivitySpanId=*Activity.SpanId" +
+                    ";ActivityTraceId=*Activity.TraceId" +
+                    ";ActivityParentSpanId=*Activity.ParentSpanId" +
+                    ";ActivityIdFormat=*Activity.IdFormat" +
+                "\r\n" +
+                "Microsoft.AspNetCore/Microsoft.AspNetCore.Hosting.HttpRequestIn.Stop@Activity1Stop:-" +
+                    "Response.StatusCode" +
+                    ";ActivityDuration=*Activity.Duration.Ticks" +
+                    ";ActivityId=*Activity.Id" +
+                "\r\n" +
+                "HttpHandlerDiagnosticListener/System.Net.Http.HttpRequestOut@Event:-" +
+                "\r\n" +
+                "HttpHandlerDiagnosticListener/System.Net.Http.HttpRequestOut.Start@Activity2Start:-" +
+                    "Request.RequestUri" +
+                    ";Request.Method" +
+                    ";Request.RequestUri.Host" +
+                    ";Request.RequestUri.Port" +
+                    ";ActivityStartTime=*Activity.StartTimeUtc.Ticks" +
+                    ";ActivityId=*Activity.Id" +
+                    ";ActivitySpanId=*Activity.SpanId" +
+                    ";ActivityTraceId=*Activity.TraceId" +
+                    ";ActivityParentSpanId=*Activity.ParentSpanId" +
+                    ";ActivityIdFormat=*Activity.IdFormat" +
+                    ";ActivityId=*Activity.Id" +
+                    "\r\n" +
+                "HttpHandlerDiagnosticListener/System.Net.Http.HttpRequestOut.Stop@Activity2Stop:-" +
+                    ";ActivityDuration=*Activity.Duration.Ticks" +
+                    ";ActivityId=*Activity.Id" +
+                "\r\n" +
+                "\"";
+
+        public override IList<EventPipeProvider> GetProviders()
+        {
+            var providers = new List<EventPipeProvider>()
```

Comment:
I verified that this is not used for logging. However, I think there's another issue here because I only see Request.Scheme data in the nettrace.

Reply:
The filter string was doubly double quoted when sent through the pipe, somehow only causing the first event and first property of that event to actually be captured. Removing the double quotes from the filter string in the configuration fixes it.